lzy/channel-manager-api/src/main/proto/ai/lzy/v1/channel/v2/channel-manager-private.proto

```diff
@@ -0,0 +1,97 @@
+syntax = "proto3";
+
+package ai.lzy.v1.channel.v2;
+
+import "ai/lzy/v1/common/channel.proto";
+import "ai/lzy/v1/long-running/operation.proto";
+import "ai/lzy/v1/long-running/option.proto";
+import "ai/lzy/v1/common/data-scheme.proto";
+
+option java_outer_classname = "LCMPS";
+option java_package = "ai.lzy.v1.channel.v2";
+
+service LzyChannelManagerPrivate {
+  rpc Create (CreateRequest) returns (CreateResponse);
+
+  rpc Destroy (DestroyRequest) returns (ai.lzy.v1.longrunning.Operation) {
+    option (ai.lzy.v1.operation) = {
+      response: "DestroyResponse"
+      metadata: "DestroyMetadata"
+    };
+  }
+
+  rpc DestroyAll (DestroyAllRequest) returns (ai.lzy.v1.longrunning.Operation) {
+    option (ai.lzy.v1.operation) = {
+      response: "DestroyAllResponse"
+      metadata: "DestroyAllMetadata"
+    };
+  }
+
+  rpc Status (StatusRequest) returns (StatusResponse);
+
+  rpc List (ListRequest) returns (ListResponse);
+}
+
+// ============== Create ==================
+
+message CreateRequest {
+  string execution_id                                     = 1;
+  string workflow_name                                    = 2;
+  string user_id                                          = 3;
+
+  /* optional */ai.lzy.v1.common.DataScheme scheme        = 4;  // If not set, using plain DataScheme
+
+  // Initial storage peer
+  oneof storage_peer {
+    common.PeerDescription.StoragePeer initial_producer   = 5;
+    common.PeerDescription.StoragePeer initial_consumer   = 6;
+  }
+}
+
+message CreateResponse {
+  string channel_id = 1;
+}
+
+// ============== Destroy ==================
+
+message DestroyRequest {
+  string channel_id = 1;
+  string cause      = 2;
```

Comment:
s/cause/reason

empty string stands for no error, doesn't it?

Reply:
`reason` is just a human-readable string, which is used for logging purposes. To control behavior, the `force` flag can be used.